lib/assessment.js

```diff
@@ -8,28 +8,66 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
+<<<<<<< HEAD
+var min = function min(array, callback, initialValue) {
+  var previousValue;
+  if (initialValue !== undefined) {
+    previousValue = initialValue;
+  } else {
+    array = array.slice();
+    previousValue = array.shift();
+  }
+
+  array.forEach(function(currentValue, index, array) {
+    previousValue = callback(previousValue, currentValue, index, array);
+  });
+
+  return previousValue;
+};
```

Comment:
This is Antony's `reduce` implementation. It's not what the question asked for; Q1 wants the function that will be passed in as `callback` in order to make it return the minimum element of an array of numbers.